### DISCUSSION TITLE
Nice, but... can it work like this also?

### DISCUSSION THREAD

**Discussion by pickonedev** [iantrich/restriction-card#46] [Ideas]
Hi!

I just wanted to say that this plugin it is very useful, I am using it only for confirmation, no block, no pin. Now, I am asking if this plugin could run the button pressed after I confirm "ok"... Can it? Or it is hard to do it?

I have some swtiches, and when I click them, I just want to get a confirmation dialog, then the switch should perform without touching it again. Like a confirmation dialog. Can this plugin do this or it can be implemented?

Thank you in advance, great work.

**Comment by iantrich** [comment]
No, it is just an overlay. It has no interaction with the element within

**Reply by ShadyMCShadesen** [reply]
but this would be awesome^^

**Reply by iantrich** [reply]
You should look at the confirmation option in core lovelace for actions. Sounds like it does what you want.
